app/authentication/auth.py

```diff
@@ -187,6 +187,8 @@ def validate_service_api_key_auth():  # noqa: C901
     auth_token = get_auth_token(request)
     client = __get_token_issuer(auth_token)
 
+    form = request.form.to_dict(flat=True)
```

Comment:
Since this is a request to a Flack application - request.form will ALWAYS be populated.